src/animation.js

```diff
@@ -23,6 +23,14 @@ const TAG_ = 'Animation';
 
 const NOOP_CALLBACK = function() {};
 
+/** @typedef {./curve.CurveDef} */
+let CurveDef;
```

Comment:
Why did you alias these instead of using the type inline?
I think that would be better.